tests/lobpcg_test.py

```diff
@@ -275,6 +275,9 @@ def _possibly_plot(self, A, eigs, X, m, matrix_name):
     if not os.getenv('LOBPCG_EMIT_DEBUG_PLOTS'):
       return
 
+    if os.getenv('LOBPCG_EMIT_DEBUG_PLOTS') == "0":
+      return
```

Comment:
I think the intent here is that if the env variable is defined (and set to any value) then the plots are skipped.

Reply:
Sure, we can stick with the previous version and add a required information in documentation